samples/CommunityToolkit.Maui.Markup.Sample/Pages/Base/BasePage.cs

```diff
@@ -5,11 +5,11 @@ namespace CommunityToolkit.Maui.Markup.Sample.Pages.Base;
 
 abstract class BaseContentPage<T> : ContentPage where T : BaseViewModel
```

Comment:
Let's rename to BasePage or rename file to BaseContentPage

Reply:
Done ✅